vochain/scrutinizer/scrutinizer.go

```diff
@@ -75,22 +62,19 @@ func (s *Scrutinizer) Commit(height int64) {
 	go s.checkFinishedProcesses(height)
 
 	// Add Entity and register new active process
-	var isLive bool
 	var err error
 	var nvotes int64
-	for _, p := range s.processPool {
-		s.addEntity(p.EntityID, p.ProcessID)
-		if isLive, err = s.isLiveResultsProcess(p.ProcessID); err != nil {
-			log.Errorf("cannot check if process is live results: (%s)", err)
-			continue
-		}
-		if isLive {
-			s.addLiveResultsProcess(p.ProcessID)
+	for _, p := range s.newProcessPool {
+		if err = s.newEmptyProcess(p.ProcessID); err != nil {
+			log.Warnf("commit: cannot create new empty process: %v", err)
 		}
 	}
 
-	for i, p := range s.resultsPool {
-		s.registerPendingProcess(p.ProcessID, height+int64(i+1))
+	// Update existing processes
+	for _, p := range s.updateProcessPool {
+		if err = s.updateProcess(p); err != nil {
```

Comment:
use `:=`

Reply:
That's the never end discussion and still not clear to me whats the most correct approach... But ok, I'll declare a new var for this error.